kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -164,18 +184,24 @@ serverPodCpuRequest: "500m"
 # serverPodCpuLimit: "1000m"
 
 #
-# RCU configuration details
+# FMW Infrastructure Domain Type. Legal values are JRF or RestrictedJRF
+fmwDomainType: JRF
+
+
+# RCU configuration details (required if fmwDomainType is 'JRF')
 #
 
 # The schema prefix to use in the database, for example `SOA1`.  You may wish to make this
 # the same as the domainUID in order to simplify matching domains to their RCU schemas.
 rcuSchemaPrefix: domain1
 
+# Password for the RCU schema
+rcuSchemaPassword: 
```

Comment:
Is the rcuSchemaPassword passed on the 'create' command line just like the WL user & pass?  If so, maybe it shouldn't be mentioned here (just like the WL user & pass).

Reply:
not currently. Should we remove this from inputs yaml file and add it to a new create-domain.sh option?

Reply:
hmmm. I suspect it should be treated the same way as other passwords. Maybe ask @jshum2479

Reply:
updated to use create-domain.sh -q to specify rcu schema password

Reply:
nice